src/app/admin/pages/learning-objects/learning-objects.component.scss

```diff
@@ -0,0 +1,79 @@
+@import '~_vars.scss';
+
+.list {
+    max-width: $max-width;
+    position: relative;
+    margin: auto;
+    box-sizing: border-box;
+    margin-top: 20px;
+    min-height: 500px;
+    overflow: hidden;
+    margin-top: 40px;
+    margin-bottom: 20px;
+}
+
+.list__body {
+    .list__body__column-headers,
+    ::ng-deep .row-item {
+        padding: 0 20px;
+        display: grid;
+        grid-template-columns: 20px 400px 2fr 1fr 1fr 50px;
+        grid-gap: 0 60px;
+        align-items: center;
+    }
+
+    ::ng-deep .row-item {
```

Comment:
Can we add a FIXME that the ng-deep should be removed in favor of a shared stylesheet?